parley/src/layout/cluster.rs

```diff
@@ -44,13 +44,17 @@ impl<'a, B: Brush> Cluster<'a, B> {
             path.line_index = line_index as u32;
             let mut offset = line.metrics().offset;
             let last_run_index = line.len().saturating_sub(1);
-            for (run_index, run) in line.runs().enumerate() {
+            for run_index in 0..line.len() {
```

Comment:
Unfortunately we can't just use `line.items()`, as `GlyphRunIter` currently appears to skip the run on empty lines.